### ISSUE TITLE
Add Granted By

### ISSUE THREAD

**Issue by masterbater** [JosephSilber/bouncer#655]
Add who granted the ability, it could nullable

**Comment by lrljoe** [comment]
That's really something you should handle separately, e.g. through an activity log or similar.

**Comment by masterbater** [comment]
> That's really something you should handle separately, e.g. through an activity log or similar.

You are right using like spatie activity, just need to use custom models and use trait. can be manually logged too.
